plugins/gcloud/gcloud.plugin.sh

```diff
@@ -0,0 +1,49 @@
+#! bash oh-my-bash.module
+# gcloud.plugin.sh
+# Author: Antonino Cangialosi (github.com/ninoCan)
```

Comment:
The original code is a part of Oh My Zsh, which is licensed with MIT License, so you need to retain the original copyright notice. The MIT License isn't meant to waive the copyright.
```suggestion
# Author: Ian Chesal (github.com/ianchesal) -- ohmyzsh/gloud.plugin.zsh
# Author: Antonino Cangialosi (github.com/ninoCan)
```

Reply:
Oh, thanks! I didn't pay attention to the license. My bad!